internal/provider/file/file.go

```diff
@@ -160,15 +162,15 @@ func (p *Provider) Start(ctx context.Context) error {
 	}
 }
 
-func (p *Provider) startHealthProbeServer(ctx context.Context, readyzChecker healthz.Checker) {
+func (p *Provider) startHealthProbeServer(ctx context.Context, readyzChecker healthz.Checker, port int) {
```

Comment:
why not directly use `p.healthzServerPort`?

Reply:
My bad! Will fix.